service-impl/data-service-impl/hdfs-data-service/src/main/java/uk/gov/gchq/palisade/data/service/reader/HDFSDataReader.java

```diff
@@ -40,34 +40,34 @@
 import static java.util.Objects.requireNonNull;
 
 /**
- * An HdfsDataReader is an implementation of {@link SerialisedDataReader} for {@code HDFS}
+ * An HDFSDataReader is an implementation of {@link SerialisedDataReader} for {@code HDFS}
  * that opens a file and returns a single {@link InputStream} containing all the records.
  */
-public class HdfsDataReader extends SerialisedDataReader {
+public class HDFSDataReader extends SerialisedDataReader {
     @JsonIgnore
     private FileSystem fs;
 
-    public HdfsDataReader() {
+    public HDFSDataReader() {
     }
 
     @JsonCreator
-    public HdfsDataReader(@JsonProperty("conf") final Map<String, String> conf,
+    public HDFSDataReader(@JsonProperty("conf") final Map<String, String> conf,
```

Comment:
I thought the json property tags were only required if the name used is different to the parameter name?